google/ggrc-core
# GGRC-5685 Show 'Open' btn in Global Search for global users

# Issue description

"Open" button is missed for global Creator, Reader, Editor.

# Steps to test the changes

1. Log to GGRC as Global Creator 
2. Open Global Search 
3. Search by any object, e.g. Control 
4. Look at the Search results
Expected Result: Global Search - "Open" button should be presented for Creator, Reader, Editor

# Solution description

The solution is to remove 'view_object_page' condition from the template, because all users who can see the list of objects in the search results have enough permissions to see the 'Open' button.

# Sanity checklist


# PR Review checklist
